echopype/convert/parse_azfp.py

```diff
@@ -197,6 +197,25 @@ def _compute_battery(self, ping_num, battery_type):
 
         return N * USL5_BAT_CONSTANT
 
+    def _computer_pressure(self, ping_num, is_valid):
+        """
+        Compute pressure in decibar
+
+        Parameters
+        ----------
+        ping_num
+            ping number
+        is_valid
+            whether the associated parameters have valid values
+        """
+        if self.parameters["sensors_flag_pressure_sensor_installed"] == "no" or not is_valid:
```

Comment:
Is the parameter "sensors_flag_pressure_sensor_installed" always present? In case it isn't, I suggest reversing the elements in this test:
```suggestion
        if not is_valid or self.parameters["sensors_flag_pressure_sensor_installed"] == "no":
```
Or maybe the "sensors_flag_pressure_sensor_installed" test isn't needed and may be misleading, since `pressure_is_valid` already carries the information we need: if "a0" or "a1" are not present or valid, the pressure computation should not be carried out.

Reply:
I think the parameter `sensors_flag_pressure_sensor_installed` will always exist. The test will make sure that even if `a0` and `a1` are present but if sensor flag is not installed, then the values are incorrect.

Reply:
Thanks. I've looked at a sample XML file that doesn't have pressure data, and you're right, it does have this XML element:
```xml
<SensorsFlag PressureSensorInstalled="no" ParosInstalled="no">0</SensorsFlag>
```